platform/kube/inject/inject.go

```diff
@@ -141,16 +133,16 @@ func ProxyImageName(hub string, tag string, debug bool) string {
 // Params describes configurable parameters for injecting istio proxy
 // into kubernetes resource.
 type Params struct {
-	InitImage         string                  `json:"initImage"`
-	ProxyImage        string                  `json:"proxyImage"`
-	Verbosity         int                     `json:"verbosity"`
-	SidecarProxyUID   int64                   `json:"sidecarProxyUID"`
-	Version           string                  `json:"version"`
-	EnableCoreDump    bool                    `json:"enableCoreDump"`
-	DebugMode         bool                    `json:"debugMode"`
-	Mesh              *proxyconfig.MeshConfig `json:"-"`
-	MeshConfigMapName string                  `json:"meshConfigMapName"`
-	ImagePullPolicy   string                  `json:"imagePullPolicy"`
+	InitImage       string                  `json:"initImage"`
+	ProxyImage      string                  `json:"proxyImage"`
+	Verbosity       int                     `json:"verbosity"`
+	SidecarProxyUID int64                   `json:"sidecarProxyUID"`
+	Version         string                  `json:"version"`
+	EnableCoreDump  bool                    `json:"enableCoreDump"`
+	DebugMode       bool                    `json:"debugMode"`
+	Mesh            *proxyconfig.MeshConfig `json:"-"`
+	MeshName        string                  `json:"meshName"`
```

Comment:
What is a MeshName?

Reply:
The name of MeshConfig CRD in istio installation namespace.